app/javascript/styles/mastodon/admin.scss

```diff
@@ -1315,12 +1315,18 @@ a.sparkline {
     &__key {
       font-weight: 500;
       padding: 11px 10px;
+      white-space: nowrap;
```

Comment:
The style changes aren't mentioned in the PR description -- are they intentionally here? (if so, before/after screenshot or separate PR?)